github-issue-summary-to-email/main.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/github;
+import wso2/choreo.sendemail as email;
```

Comment:
Why do we use choreo send mail? Can't we use `ballerina/email` module?

Reply:
Choreo sendmail was used for the ease of testing these examples for users. Compared to the ballerina email connector where a host need to be setup separately, choreo.sendmail can be used to send the email without any additional effort.